xml/operations-compute-gpu_passthrough.xml

```diff
@@ -0,0 +1,298 @@
+        the &clm; model files to enable GPU passthrough for compute nodes.
+      </para>
+      <para>
+        <emphasis role="bold">Edit servers.yml</emphasis>
+      </para>
+      <para>
+        Add the pass-through section after the definition of servers section in the yaml file.
+        Here's an example: For clarity purpose, the example shows only the relevant sections.
+      </para>
+      <screen>
+
+        ---
+        product:
+        version: 2
+
+        baremetal:
+        netmask: 255.255.255.0
+        subnet: 192.168.100.0
+
+
+        servers:
+        .
+        .
+        .
+        .
+
+          - id: compute-0001
+            ip-addr: 192.168.75.5
+            role: COMPUTE-ROLE
+            server-group: RACK3
+            nic-mapping: HP-DL360-4PORT
+            ilo-ip: ****
+            ilo-user: ****
+            ilo-password: ****
+            mac-addr: ****
+          .
+          .
+          .
+
+          - id: compute-0008
+            ip-addr: 192.168.75.7
+            role: COMPUTE-ROLE
+            server-group: RACK2
+            nic-mapping: HP-DL360-4PORT
+            ilo-ip: ****
+            ilo-user: ****
+            ilo-password: ****
+            mac-addr: ****
+
+        pass-through:
+          servers:
+            - id: compute-0001
+              data:
+                gpu:
+                  - vendor_id: 10de
+                    product_id: 1db4
+                    bus_address: 0000:84:00.0
+                    pf_mode: type-PCI
+                    name: a1
+                  - vendor_id: 10de
+                    product_id: 1db4
+                    bus_address: 0000:85:00.0
+                    pf_mode: type-PCI
+                    name: b1
+            - id: compute-0008
+              data:
+                gpu:
+                  - vendor_id: 10de
+                    product_id: 1db4
+                    pf_mode: type-PCI
+                    name: c1
+      </screen>
+      <para>
+        To make the edits,
+      </para>
+      <procedure>
+        <step>
+          <para>
+            Check out the site branch of the local git repository and change to the
+            correct directory:
+          </para>
+          <screen>
+            &prompt.ardana;cd ~/openstack
+            &prompt.ardana;git checkout site
+            &prompt.ardana;cd ~/openstack/my_cloud/definition/data/
+          </screen>
+        </step>
+        <step>
+          <para>
+            Open the file containing the servers list, e.g. servers.yml
+            with vim or another editor and make the necessary changes.
+            Save the changes to the file, then commit to the local git repository:
+          </para>
+          <screen>
+            &prompt.ardana;git add -A
+          </screen>
+          <para> Confirm that the changes to the tree are relevant changes and commit</para>
+          <screen>
+            &prompt.ardana; git status
```

Comment:
Should there be a space before git here? Does it matter?

Reply:
Removed the extra space in the latest commit